test/automation/src/positron/fixtures/positronRFixtures.ts

```diff
@@ -30,7 +30,7 @@ export class PositronRFixtures {
 		// We currently don't capture fixtures in the Playwright trace, so take a screenshot on failure
 		try {
 			await this.app.workbench.positronConsole.selectInterpreter(InterpreterType.R, desiredR, skipReadinessCheck);
-			await this.app.workbench.positronConsole.waitForReady('>', 2000);
+			await this.app.workbench.positronConsole.waitForReady('>', 40000);
```

Comment:
Oh wow.  Was such a huge bump needed?  2 -> 40 s

Reply:
I don't think it was actually using the timeout if I recall. I think it was a typo, a 2 second wait???

Reply:
updated per our convo 👍